attributes/default.rb

```diff
@@ -45,6 +47,17 @@
   zk_announce_path: '/aurora/svc'
 }
 
+default['aurora']['client'] = [
+  {
+    auth_mechanism: "UNAUTHENTICATED",
+    name: "test-cluster",
+    scheduler_zk_path: "/aurora/scheduler",
+    slave_root: "/var/lib/mesos",
+    slave_run_directory: "latest",
+    scheduler_uri: "http://localhost:8082",
+  }
+]
+
```

Comment:
Should we use the `zookeeper` version of this config (http://aurora.apache.org/documentation/0.12.0/client-commands/#cluster-configuration) ? Seems more convenient for most setups that will be HA.

BTW, you're providing some options related to the ZK setup (`scheduler_zk_path` for ex.)